chromadb/server/fastapi/__init__.py

```diff
@@ -140,6 +141,7 @@ def __init__(self, settings: Settings):
             allow_origins=settings.chroma_server_cors_allow_origins,
             allow_methods=["*"],
         )
+        self._app.add_exception_handler(QuotaError, self.quota_exception_handler)
```

Comment:
Why is a dedicated exception handler better than using the existing `catch_exceptions_middleware` to handle the error?

Reply:
I did not see it, can be part of another refactoring. Or maybe we should use add_exception_handler which is the way of doing it in fastapi